.github/workflows/ci.yml

```diff
@@ -0,0 +1,68 @@
+name: CI
+
+on:
+  pull_request: ~
+  push: ~
+
+jobs:
+  phpunit:
+    name: PHPUnit
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 10
+      matrix:
+        run:
+          - php: '7.4'
+            sf_version: '4.4.*'
+          - php: '8.0'
+            sf_version: '5.3.*'
+          - php: '7.1'
+            sf_version: '*'
+          - php: '7.2'
+            sf_version: '*'
+          - php: '7.3'
+            sf_version: '*'
+          - php: '7.4'
+            sf_version: '*'
+
+    steps:
+      - name: Set up PHP
+        uses: shivammathur/setup-php@2.7.0
```

Comment:
What about using `shivammathur/setup-php@^2`? This way we wouldn't have to care about the action's version in the future for each bug fix/new features.

Reply:
I prefer to specify the version so nothing unexpected changes. Currently we don’t have to care about future bugs in setup-php. 

If we want to use a new feature, we can bump the version then.